src/components/Home/WalletCard.tsx

```diff
@@ -32,24 +39,37 @@ export function WalletCard({ type, onClick, ...props }: WalletCardProps) {
         return (
             <motion.div className="h-full">
                 <Card
-                    className="h-full min-w-[300px] rounded-md text-black hover:cursor-pointer"
+                    className="h-full min-w-[300px] rounded-xl bg-purple-4/20 text-black hover:cursor-pointer"
```

Comment:
thought: we prob should rename our colors to primary, secondary etc

Reply:
@Hugo0 agreed, for this my suggestion would be to fix all the colors separately, after certain features are done. This is a small task but will need some time. Happy to tackle this on another spirint :)